src/rshim.h

```diff
@@ -184,6 +184,8 @@ enum {
 #define RSHIM_YU_BASE_ADDR             0x2800000
 #define RSHIM_YU_BOOT_RECORD_OPN       0xfd8
 #define RSHIM_YU_BOOT_RECORD_OPN_SIZE  16
+#define RSHIM_YU_BF3_BASE_ADDR         0x3400000
```

Comment:
This is unused. Why do we need to add it?

Reply:
Removed.  I was trying to clean up bf3_rshim_pcie_chan_map[] in rshim_pcie.c to use offset calculation, but don't want to put too much unrelated changes in this commit. Maybe revisit it later.